index.html

```diff
@@ -4,64 +4,68 @@
     <meta charset="UTF-8">
     <title>Fork Me! FCC: Test Suite Template</title>
     <link rel="stylesheet" href="css/style.css">
+     <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+      <script  src="js/index.js"></script>
   </head>
 
   <body>
-    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
-     <nav id ="navbar" >
-       <header>this documentation is a sum of all of mankind knowledge</header>
```

Comment:
## You are doing a good job thus far
- I think it will be better to use h1/h2/h3 instead of the header tag

Reply:
- [ ] Why do you think it would be better to use `<hx>` tags? When giving a suggestion, kindly explain why you think a student should make those changes. This helps them to understand better and learn. 👍